src/app/parcels/batch/displayComponents/PersonInput.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+    getAdultBirthYears,
+    getChildBirthYears,
+    getCurrentYearChildBirthMonthList,
+} from "@/app/clients/form/birthYearDropdown";
+import { UncontrolledSelect } from "@/components/DataInput/DropDownSelect";
+import { Gender, Person } from "@/components/Form/formFunctions";
+import { GappedDiv } from "@/components/Form/formStyling";
+import styled from "styled-components";
+import { PersonField } from "@/app/parcels/batch/displayComponents/PersonEditCell";
+import { Divider } from "@mui/material";
+import { BatchActionType } from "@/app/parcels/batch/batchTypes";
+import dayjs from "dayjs";
+
+interface PersonInputProps {
+    people: Person[];
+    setPeople: React.Dispatch<React.SetStateAction<Person[]>>;
+    index: number;
+    personField: PersonField;
+    dispatchBatchTableAction: React.Dispatch<BatchActionType>;
+    id: number;
+    dispatchPerson: (newPerson: Person[]) => void;
+}
+
+const GappedDivTopMargin = styled(GappedDiv)`
+    margin-top: 0.5rem;
+    padding: 0.5rem;
+`;
+
+const PersonInput: React.FC<PersonInputProps> = ({
+    people,
+    setPeople,
+    index,
+    personField,
+    dispatchBatchTableAction,
+    id,
+    dispatchPerson,
+}) => {
+    const yearLabelsAndValues: [string, string][] =
+        personField === "adultInfo"
+            ? getAdultBirthYears().map((year) => [`${year}`, `${year}`] as [string, string])
+            : getChildBirthYears().map((year) => [`${year}`, `${year}`] as [string, string]);
+    const genderLabelsAndValues: [string, string][] = [
+        ["Male", "male"],
+        ["Female", "female"],
+        ["Prefer Not To Say", "other"],
+    ];
+    const defaultMonthString = people[index].birthMonth
+        ? dayjs(people[index].birthMonth).month() + 1
+        : "";
```

Comment:
This feels weird. It's called "string" but it looks like it can be a number. I also don't fully understand what the `dayjs` call is for, does that convert between e.g., `"April"` and `3`?